__tests__/app.test.js

```diff
@@ -44,6 +44,44 @@ describe("/api", () => {
   });
 });
 
+describe("/api/articles/:article_id", () => {
+  test("GET:200 responds with an article object", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        const article = body.article;
+        expect(article).toMatchObject({
```

Comment:
Given that we know the exact expected values here, the test should reflect that. E.g. if requesting to get the article with the id of 1 like here, we wouldn't expect back **any** number for article_id - we'd expect the article_id to be 1! Same for the other vals